### DISCUSSION TITLE
Wrong Taskbar Icon

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#832] [Ideas]
### What happened?

Hey, just installed Zen from AUR and when I open it the opened-windows icon is different from Zen's one as you can see in the screenshot:
![Screenshot_20240822_204815](https://github.com/user-attachments/assets/a4854497-f259-4cf2-a392-7127897499e5)

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0-a.27

### Severity impact

Enhancement

### What platform are you seeing the problem on?

Linux

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/449_

**Comment by SomeDudeBrosky** [comment]
This is a bug and is for the maintainer of the AUR as that is an unofficial package. The only official package for Linux is a flatpak.

**Reply by anvnh** [reply]
i tried to install the flatpak version, but seem like it doesn't work.

**Comment by NTICompass** [comment]
I'm on Fedora 41, using the "unofficial" package from [here](https://copr.fedorainfracloud.org/coprs/sneexy/zen-browser/) and I am having this issue (again) on the newly released `1.6b`.  The icon was correct in the previous `1.0.2-b.5`, but now it's broken.  I don't know if this is an issue with zen or the package maintainer, but it's weird that it was working and now it's not.

![Screenshot_20250110_075350](https://github.com/user-attachments/assets/07470ab6-5eb0-49de-81fa-3975a4b80a35)

**Comment by verjax** [comment]
What worked for me was recreating the browser entry and match it to the browser name as per my install, then pin that to my dash:


rm ~/.local/share/applications/zen-browser.desktop

cat > ~/.local/share/applications/zen-browser.desktop << 'EOF'
[Desktop Entry]
Version=1.0
Name=Zen Browser
Comment=Experience tranquillity while browsing the web
GenericName=Web Browser
Keywords=Internet;WWW;Browser;Web;Explorer
Exec=/home/[user]/opt/zen/zen %u
StartupWMClass=zen
Terminal=false
Type=Application
Icon=/home/[user]/opt/zen/browser/chrome/icons/default/default128.png
Categories=Network;WebBrowser;
MimeType=text/html;text/xml;application/xhtml+xml;x-scheme-handler/http;x-scheme-handler/https;
StartupNotify=true
EOF

update-desktop-database ~/.local/share/applications/
